### ISSUE TITLE
Can't install OpenBoard on lubuntu

### ISSUE THREAD

**Issue by ghost** [OpenBoard-org/OpenBoard#173]
I tried to install it on lubuntu 18.04 but it fails
I also tried to install Qt 5.5 but repo does not exist, 404 error

```
@optiplex745-ubuntu:~/Hentet$ sudo dpkg -i openboard.deb
Vælger tidligere fravalgt pakke openboard.
(Læser database ... 139081 filer og kataloger installeret i øjeblikket.)
Gør klar til at udpakke openboard.deb ...
Udpakker openboard (1.4.1) ...
dpkg: afhængighedsproblemer forhindrer konfiguration af openboard:
 openboard afhænger af libavformat-ffmpeg56 (>= 7:2.8.14); men:
  Pakken 'libavformat-ffmpeg56' er ikke installeret.
 openboard afhænger af libavcodec-ffmpeg56 (>= 7:2.8.14) | libavcodec-ffmpeg-extra56 (>= 7:2.8.14); men:
  Pakken 'libavcodec-ffmpeg56' er ikke installeret.
  Pakken 'libavcodec-ffmpeg-extra56' er ikke installeret.
 openboard afhænger af libswresample-ffmpeg1 (>= 7:2.8.14); men:
  Pakken 'libswresample-ffmpeg1' er ikke installeret.
 openboard afhænger af libswscale-ffmpeg3 (>= 7:2.8.14); men:
  Pakken 'libswscale-ffmpeg3' er ikke installeret.
 openboard afhænger af libavutil-ffmpeg54 (>= 7:2.8.14); men:
  Pakken 'libavutil-ffmpeg54' er ikke installeret.
 openboard afhænger af libqt5webkit5 (>= 5.5.1); men:
  Pakken 'libqt5webkit5' er ikke installeret.
 openboard afhænger af libqt5multimediawidgets5 (>= 5.5.1); men:
  Pakken 'libqt5multimediawidgets5' er ikke installeret.
 openboard afhænger af libqt5printsupport5 (>= 5.5.1); men:
  Pakken 'libqt5printsupport5' er ikke installeret.
 openboard afhænger af libqt5multimedia5 (>= 5.5.1); men:
  Pakken 'libqt5multimedia5' er ikke installeret.
 openboard afhænger af libqt5xml5 (>= 5.5.1); men:
  Pakken 'libqt5xml5' er ikke installeret.
 openboard afhænger af libqt5script5 (>= 5.5.1); men:
  Pakken 'libqt5script5' er ikke installeret.
 openboard afhænger af onboard; men:
  Pakken 'onboard' er ikke installeret.
 openboard afhænger af libqt5multimedia5-plugins; men:
  Pakken 'libqt5multimedia5-plugins' er ikke installeret.

dpkg: fejl under behandling af pakken openboard (--install):
 afhængighedsproblemer - efterlader den ukonfigureret
Behandler udløsere for desktop-file-utils (0.23-1ubuntu3.18.04.1) ...
Behandler udløsere for mime-support (3.60ubuntu1) ...
Der opstod fejl under behandlingen:
 openboard
uldum@optiplex745-ubuntu:~/Hentet$ 

```

**Comment by Arvil** [comment]
Hello,

Have you installed the list of packages that dpkg tells you are not installed ?

Cheers

**Comment by evildog1** [comment]
I tried
```
sudo apt-get update
sudo apt-get install lubuntu-restricted-extras
sudo apt-get install lubuntu-restricted-addons
```
Still missing packages

**Comment by audimax85** [comment]
Hi,
I had the same Problem on Ubuntu 18.04.
Openboard wants a new(er) Version of ffmpeg56 an bionic has ffmpeg57.
I solved it by adding the xenial (16.4) repos to /etc/apt/sources.list

`deb http://de.archive.ubuntu.com/ubuntu/ xenial main universe
deb http://de.archive.ubuntu.com/ubuntu/ xenial-updates main universe`

Afterwards it installed just fine.

**Comment by Arvil** [comment]
Are you trying to install a 16.04 .deb on Ubuntu xenial ? There are xenial builds out there in other issues that should do the trick...

**Comment by evildog1** [comment]
@audimax85 thanks, it works now

**Comment by profejuanca** [comment]
The libavformat-ffmpeg56 is obsolete and no longer available in repos on 18.04, It's possible to port the package or code for libavformat-ffmpeg57? many programs use  this and itś incompatible to install both.
